notebooks/fit_gaussian_mlp_ensemble_1d.ipynb

```diff
@@ -215,4 +215,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 2
-}
+}
```

Comment:
nitpick: this doesn't seem intended

Reply:
I think this was the IDE's decision, but It's loading properly in Github right now so I'll leave it like this :P